config/routes.rb

```diff
@@ -16,6 +16,12 @@
     root to: "auth/sessions#new"
   end
 
+  get '/installer/options', to: 'installer#options'
+  get '/installer/master', to: 'installer#master'
+  get '/nodes/add', to: 'nodes#add'
+  get '/nodes', to: 'nodes#list'
+  get '/config', to: 'config#index'
```

Comment:
Instead of `nodes#list`, please use `nodes#index`, which is the REST-way of handling this. For `nodes#add`, you should go for `nodes#new`. The installer paths can also be RESTified.

That being said, that's something we can fix after this review.